keps/sig-network/3705-cloud-node-ips/README.md

```diff
@@ -600,53 +290,30 @@ to start after downgrade.
 ###### How can this feature be enabled / disabled in a live cluster?
 
 - [X] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: CloudNodeIPs
+  - Feature gate name: CloudDualStackNodeIPs
   - Components depending on the feature gate:
     - kubelet
     - cloud-controller-manager
 
 ###### Does enabling the feature change any default behavior?
 
-Kubelet will begin setting two annotations on each node rather than
-one. However, cloud providers will still prefer the old annotation and
-will only read the new annotation if the old one isn't there.
-
-So assuming no bugs, the only visible change after (just) enabling the
-feature is the duplicate annotation.
+No
```

Comment:
Isn't https://github.com/kubernetes/enhancements/pull/3898/files effectively a change in default behavior?

Reply:
Ok - I guess you're dropping the proposal from that PR, right?
Assuming I'm right here, I actually like it especially given backward compatibility here...

Reply:
I'm not sure which part of that PR you're referring to, but it was never proposing a change in default behavior.

The original KEP-3705 said that we would
1. add dual-stack `--node-ips` for external clouds
2. add `--node-ip IPv4` and `--node-ip IPv6` etc
3. rename the node IP annotation

#3898 just dropped parts 2 and 3 from alpha, and now this PR drops them from beta and GA too.

There was some discussion in #3898 about stuff we _might_ do in the future, but even then, the assumption was that we'd do it in a backward-compatible way (eg, leave `--node-ip` with its current bad semantics and add `--node-ips` with new-and-improved semantics.)

Reply:
OK - I guess I was too much biased by the "late-breaking change" header.  I've read it more carefully now and it all makes sense now.